amd_openvx_extensions/amd_rpp/source/AbsoluteDifference.cpp

```diff
@@ -105,6 +105,7 @@ static vx_status VX_CALLBACK validateAbsoluteDifference(vx_node node, const vx_r
 static vx_status VX_CALLBACK processAbsoluteDifference(vx_node node, const vx_reference * parameters, vx_uint32 num) 
 { 
 	RppStatus status = RPP_SUCCESS;
+	vx_status return_status = VX_SUCCESS;
```

Comment:
The name return_status and status are confusing. It would be better to use  more meaningful name 
RppStatus rpp_status
vx_status status
Also make sure RppStatus return value is negative for failed return codes